utils_test.go

```diff
@@ -36,12 +36,10 @@ var dummyData = map[string]interface{}{
 		"tags":    []string{"Python", "Fortran", "Lisp"},
 	},
 	"collection1": map[string]interface{}{
-		"name":   "first collection",
-		"userId": 1,
+		"name": "first collection",
 	},
 	"collection2": map[string]interface{}{
-		"name":   "second collection",
-		"userId": 1,
+		"name": "second collection",
```

Comment:
@mikey2020 In the dummy, there are four test collections. It's not paramount but I think we can do a bit of clean-up there 'cause we clearly don't need them all